scripts/monitoring/cron-send-project-operation.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+""" Create project check for v3 """
+
+import argparse
+import datetime
+import time
+
+#pylint: disable=import-error
+from openshift_tools.monitoring.ocutil import OCUtil
+from openshift_tools.monitoring.metric_sender import MetricSender
+
+import logging
+logging.basicConfig(
+    format='%(asctime)s - %(relativeCreated)6d - %(levelname)-8s - %(message)s',
+)
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+ocutil = OCUtil()
+
+commandDelay = 5 # seconds
+
+def runOCcmd(cmd, base_cmd='oc'):
+    """ log commands through ocutil """
+    logger.info(base_cmd + " " + cmd)
+    oc_time = time.time()
+    oc_result = ocutil.run_user_cmd(cmd, base_cmd=base_cmd, )
+    logger.info("oc command took %s seconds", str(time.time() - oc_time))
+    return oc_result
+
+def parse_args():
+    """ parse the args from the cli """
+    logger.debug("parse_args()")
+
+    parser = argparse.ArgumentParser(description='OpenShift app create end-to-end test')
+    parser.add_argument('-v', '--verbose', action='store_true', default=None, help='Verbose?')
+    parser.add_argument('--namespace', default="ops-project-operation-check",
+                        help='namespace (be careful of using existing namespaces)')
+    return parser.parse_args()
+
+def send_metrics(status_code_create, status_code_delete):
+    """ send data to MetricSender"""
+    logger.debug("send_metrics()")
+
+    ms_time = time.time()
+    ms = MetricSender()
+    logger.info("Send data to MetricSender")
+
+    # 1 means create and delete the project failed
+    ms.add_metric({'openshift.master.project.create': status_code_create})
+    ms.add_metric({'openshift.master.project.delete': status_code_delete})
+
+    ms.send_metrics()
+    logger.info("Data sent to Zagg in %s seconds", str(time.time() - ms_time))
+
+def check_project(config):
+    """ check create and delete project """
+    logger.info('check_project()')
+    logger.debug(config)
+
+    project = None
+
+    try:
+        project = runOCcmd("get project {}".format(config.namespace))
+        logger.debug(project)
+    except Exception:
+        pass # don't want exception if project not found
+    if project:
+        project_exist = 0 # project exists
+    else:
+        project_exist = 1 # project doest not exists
+
+    return project_exist
+
+def create_project(config):
+    " create the project "
+    try:
+        runOCcmd("new-project {}".format(config.namespace), base_cmd='oadm')
+        time.sleep(commandDelay)
+        project = runOCcmd("get project {}".format(config.namespace))
+    except Exception:
+        logger.exception('error creating new project')
+
+def delete_project(config):
+    " delete the project "
+    try:
+        runOCcmd("delete project {}".format(config.namespace), base_cmd='oc')
+        time.sleep(commandDelay)
+    except Exception:
+        logger.exception('error delete project')
+def main():
```

Comment:
code flair - please add a single newline between the end of one function and the beginning of another